plugins/org.jboss.tools.quarkus.core/src/org/jboss/tools/quarkus/core/project/ClasspathContainersHelper.java

```diff
@@ -0,0 +1,37 @@
+/*************************************************************************************
```

Comment:
Other than a since tag addition, I don't believe this file was changed and so I don't think it'd really be appropriate to update the copyright date. But... I guess I can.

Reply:
ah ok it's a plain copy/paste, not a new class. gotcha.
still, has it's been added today, copyright is now ok for me.